### PR TITLE
Add docs index

### PR DESCRIPTION
**Pull request by austinlparker**
Add an index page to the docs folder so we can more easily link to it.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+*Running the Demo*
+Want to deploy the demo and see it in action? Start here.
+
+- [Docker](./docker_deployment.md)
+- [Kubernetes](./kubernetes_deployment.md)
+
+*Service Documentation*
+Specific information about how OpenTelemetry is deployed in each service can be
+found here.
```
Comment (reyang) [inline docs/README.md:17]: minor, maybe:

```suggestion
found here:
```

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- [Recommendation Service](./services/recommendationservice.md)
+- [Quote Service](./services/quoteservice.md)
+- [Shipping Service](./services/shippingservice.md)
+
+*Scenarios*
+How can you solve problems with OpenTelemetry? These scenarios walk you through
+some pre-configured problems and show you how to interpret OpenTelemetry data to
+solve them.
+
+- [NYI](./README.md)
```
Comment (reyang) [inline docs/README.md:35]: What is `NYI`?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+*Reference*
```
Comment (reyang) [inline docs/README.md:37]:
```suggestion
## Reference
```

**Review by reyang — APPROVED** [review]
LGTM with minor, non-blocking suggestions.

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# OpenTelemetry Demo Documentation
+
+Welcome to the OpenTelemetry Demo! This folder contains overall documentation
+for the project, how to install and run it, and some scenarios you can use to
+view OpenTelemetry in action.
+
+## Table of Contents
+
+*Running the Demo*
```
Comment (julianocosta89) [inline docs/README.md:9]:
```suggestion
### Running the Demo
```

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Want to deploy the demo and see it in action? Start here.
+
+- [Docker](./docker_deployment.md)
+- [Kubernetes](./kubernetes_deployment.md)
+
+*Service Documentation*
```
Comment (julianocosta89) [inline docs/README.md:15]:
```suggestion
### Service Documentation
```

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- [Checkout Service](./services/checkoutservice.md)
+- [Email Service](./services/emailservice.md)
+- [Feature Flag Service](./services/featureflagservice.md)
+- [Payment Service](./services/paymentservice.md)
+- [Product Catalog Service](./services/productcatalogservice.md)
+- [Recommendation Service](./services/recommendationservice.md)
+- [Quote Service](./services/quoteservice.md)
+- [Shipping Service](./services/shippingservice.md)
+
+*Scenarios*
```
Comment (julianocosta89) [inline docs/README.md:30]:
```suggestion
### Scenarios
```

File: docs/README.md (hunk 3 above)
Comment (julianocosta89) [inline docs/README.md:37]:
```suggestion
### Reference
```

**Review by julianocosta89 — APPROVED** [review]
I've just added some headings to have a better visual feeling.
But the content is great!

**Comment by cartersocha** [comment]
Markdown lint is failing due to the headings needing a space in between the following text. I think we're good besides that for now besides @reyang's outstanding "NYI" question

File: docs/README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# OpenTelemetry Demo Documentation
+
+Welcome to the OpenTelemetry Demo! This folder contains overall documentation
+for the project, how to install and run it, and some scenarios you can use to
+view OpenTelemetry in action.
+
+## Table of Contents
+
+### Running the Demo
```
Comment (julianocosta89) [inline docs/README.md:9]:
```suggestion
### Running the Demo

```

File: docs/README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+Want to deploy the demo and see it in action? Start here.
+
+- [Docker](./docker_deployment.md)
+- [Kubernetes](./kubernetes_deployment.md)
+
+### Service Documentation
```
Comment (julianocosta89) [inline docs/README.md:15]:
```suggestion
### Service Documentation

```

File: docs/README.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+- [Checkout Service](./services/checkoutservice.md)
+- [Email Service](./services/emailservice.md)
+- [Feature Flag Service](./services/featureflagservice.md)
+- [Payment Service](./services/paymentservice.md)
+- [Product Catalog Service](./services/productcatalogservice.md)
+- [Recommendation Service](./services/recommendationservice.md)
+- [Quote Service](./services/quoteservice.md)
+- [Shipping Service](./services/shippingservice.md)
+
+### Scenarios
```
Comment (julianocosta89) [inline docs/README.md:30]:
```suggestion
### Scenarios

```

File: docs/README.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+How can you solve problems with OpenTelemetry? These scenarios walk you through
+some pre-configured problems and show you how to interpret OpenTelemetry data to
+solve them.
+
+- [NYI](./README.md)
+
+### Reference
```
Comment (julianocosta89) [inline docs/README.md:37]:
```suggestion
### Reference

```

**Review by julianocosta89 — APPROVED** [review]

File: docs/README.md (hunk 2 above)
Comment (austinlparker) [reply docs/README.md:35]: not yet implemented

**Pull request merged**